plugin.video.youtube/resources/lib/youtube/helper/yt_specials.py

```diff
@@ -51,17 +56,22 @@ def _process_browse_channels(provider, context, re_match):
     guide_id = context.get_param('guide_id', '')
     if guide_id:
         json_data = provider.get_client(context).get_guide_category(guide_id)
+        if not v3.handle_error(provider, context, json_data):
+            return False
         result.extend(v3.response_to_items(provider, context, json_data))
         pass
     else:
         json_data = provider.get_client(context).get_guide_categories()
+        if not v3.handle_error(provider, context, json_data):
+            return False
         result.extend(v3.response_to_items(provider, context, json_data))
         pass
 
     return result
 
 
 def _process_disliked_videos(provider, context, re_match):
+    provider.set_content_type(context, kodion.constants.content_type.EPISODES)
```

Comment:
> episodes fits as far as views / youtube information per video for the most part which is why I'm guessing this is what was chosen.

Plug-in authors shouldnt workaround the "issue" that some skins do not pay attention that much to the more seldom used content types. The whole point of our plug-in system is that the plug-in author should not need to know about any skin´s inner workings. These workarounds just hide the problems we need to solve in a proper way.
By using "episodes" as content type, several skins will display factually wrong information at some places. (Example: Lot of skins show "TV shows" in breadcrumbs when content is set to "episodes".)